scripts/onboarding/aks/multi-tenancy/existingClusterOnboarding.json

```diff
@@ -0,0 +1,178 @@
+        "ingestionDCE": "[if(endsWith(variables('ingestionDCEName'), '-'), substring(variables('ingestionDCEName'), 0, 42), variables('ingestionDCEName'))]",
+        "ingestionDataCollectionEndpointId": "[resourceId(variables('clusterSubscriptionId'), variables('clusterResourceGroup'), 'Microsoft.Insights/dataCollectionEndpoints', variables('ingestionDCE'))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Insights/dataCollectionEndpoints",
+            "apiVersion": "2022-06-01",
+            "name": "[variables('ingestionDCE')]",
+            "location": "[variables('workspaceLocation')]",
+            "tags": "[parameters('resourceTagValues')]",
+            "kind": "Linux",
+            "properties": {
+                "networkAcls": {
+                    "publicNetworkAccess": "Enabled"
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Resources/deployments",
+            "name": "[Concat('aks-monitoring-msi-dcr', '-',  uniqueString(variables('dcrName')))]",
+            "apiVersion": "2017-05-10",
+            "subscriptionId": "[variables('clusterSubscriptionId')]",
+            "resourceGroup": "[variables('clusterResourceGroup')]",
+            "dependsOn": [
+                "[variables('ingestionDCE')]"
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "template": {
+                    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+                    "contentVersion": "1.0.0.0",
+                    "parameters": {},
+                    "variables": {},
+                    "resources": [
+                        {
+                            "type": "Microsoft.Insights/dataCollectionRules",
+                            "apiVersion": "2022-06-01",
+                            "name": "[variables('dcrName')]",
+                            "location": "[parameters('workspaceRegion')]",
+                            "tags": "[parameters('resourceTagValues')]",
+                            "kind": "Linux",
+                            "properties": {
+                                "dataSources": {
+                                    "extensions": [
+                                        {
+                                            "name": "ContainerLogV2Extension",
+                                            "streams": [
+                                                "Microsoft-ContainerLogV2-HighScale"
+                                            ],
+                                            "extensionSettings": {
+                                                "dataCollectionSettings": {
+                                                    "namespaces": "[parameters('k8sNamespaces')]"
```

Comment:
how about
"interval": "[parameters('dataCollectionInterval')]",
                                            "namespaceFilteringMode": "[parameters('namespaceFilteringModeForDataCollection')]",
                                            "namespaces": "[parameters('namespacesForDataCollection')]",
                                            "enableContainerLogV2": "[parameters('enableContainerLogV2')]"

also here we should set enableContainerLogV2 as true as default right